.eslintrc

```diff
@@ -6,6 +6,9 @@
     //     "eslint:recommended",
     //     "plugin:react/recommended",
     // ],
+    "extends": [
+        "eslint:recommended",
+    ],
```

Comment:
Not sure if we want to break builds based on comma usage. I think we should override to not be an error. Feels like pointless friction, when you're editing a bunch of json dicts, its nice to treat the first line as the last line and leave the comma.